src/python/phenix_apps/apps/scorch/cc/cc.py

```diff
@@ -168,6 +182,36 @@ def __run(self, stage):
                         sys.exit(1)
 
 
+    def __send_cmd_as_file(self, mm, hostname, cmd):
+        cmd_file = f'run-{self.extract_run_name}_{str(uuid.uuid4())}'
```

Comment:
should be self.extract_run_name() since you want to execute the method. This is where we are seeing BOUND returned (printing the method instead of executing it).

Reply:
Ugh... I'm an idiot. Thank you for finding this.